api/src/main/java/org/openmrs/module/systemmonitor/api/db/hibernate/HibernateSystemMonitorDAO.java

```diff
@@ -119,6 +119,26 @@ public String getToday() {
 	}
 
 	@Override
+	public String getYesterdayStartDate() {
+		Calendar calendar = Calendar.getInstance(Context.getLocale());
+
+		calendar.add(Calendar.DATE, -1);
+		resetDateTimes(calendar);
+
+		return sdf.format(calendar.getTime());
+	}
+
+	@Override
+	public String getYesterdayEndDate() {
+		Calendar calendar = Calendar.getInstance(Context.getLocale());
+
+		resetDateTimes(calendar);
+		calendar.add(Calendar.MILLISECOND, -1);
```

Comment:
@kaweesi what do you think about this end date calculation? I subtracted a millisecond to make sure we don't get results at midnight today, but I see this isn't does on the other end dates. Is it not required do you think?